### PR TITLE
Feature/cas 6 service discovery

### PR DESCRIPTION
**Pull request by mbehlendorf**

### CODE REVIEW

File: app/etc/cas/config/cas.properties [hunk 1]
```diff
@@ -1 +1,22 @@
-# Is overridden by the dogu
```
Comment (Subaro) [inline app/etc/cas/config/cas.properties:1]: Currently, we fill multiple .properties files:
* resources/etc/cas/config/cas.properties.tpl
* app/etc/cas/config/cas.properties
I think you should remove these additions as they are begin overwritten when cas starts. Two property files with the same entries makes it harder to find the file that actually introduce changes to the cas.

**Review by Subaro — CHANGES_REQUESTED** [review]
The new code seems good. The old one was adapted quite well. The unit test run without any Problem. It is only missing that the unit test results are archived at the Jenkins Build.

Also SonarCube seems to to report two bugs

File: app/etc/cas/config/cas.properties (hunk 1 above)
Comment (mbehlendorf) [reply app/etc/cas/config/cas.properties:1]: The content in `app/etc/cas/config/cas.properties` is necessary for local development. So in my opinion we should keep it

Comment (Subaro) [reply app/etc/cas/config/cas.properties:1]: This is a good reason. I did not think of that.

**Pull request merged**